### PR TITLE
Added customizing User interfaces doc page

### PR DESCRIPTION
**Pull request by GDRDABARERA**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/develop/customizing-user-interfaces.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+### Customization of User Interfaces of WSO2 Identity Server
+
+With the latest release of [IS
```
Comment (ayshsandu) [inline en/docs/develop/customizing-user-interfaces.md]: remove latests, instead say from 5.9.0 release onwards

File: en/mkdocs.yml [hunk 2]
```diff
@@ -515,7 +515,8 @@ nav:
           - 'Entitlement with APIs':
             - 'Entitlement with APIs': develop/entitlement-with-apis.md
             - 'Entitlement with REST APIs': develop/entitlement-with-rest-apis.md
-      - 'Customizing User Interfaces': 
+      - 'Customizing User Interfaces':
+        - 'Customizing User Interfaces of WSO2 Identity Server': develop/customizing-user-interfaces.md
```
Comment (YvonneW) [inline en/mkdocs.yml]: Let's rephrase to 'Customizing UIs'

File: en/docs/develop/customizing-user-interfaces.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+In [WSO2 Identity Server
+5.9.0](https://wso2.com/identity-and-access-management/), all the UIs are
+present in **authenticationendpoint **and **accountrecoveryendpoint **web apps
+in <IS-Home>/repository/deployment/server/webapps folder. Please follow the
+below steps to customize these pages easily.
+
+*****
+
+#### **Step 1: Create a new extension folder inside these two web apps.**
```
Comment (YvonneW) [inline en/docs/develop/customizing-user-interfaces.md]: Use ## - correct in other applicable places as well.

File: en/docs/develop/customizing-user-interfaces.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+With the latest release of [IS
+5.9.0](https://wso2.com/identity-and-access-management/), a new UI extension has introduced to 
+easily customize the basic user interfaces like login page, username and password recovery pages, 
+single sign-on pages etc.
+
+In [WSO2 Identity Server
+5.9.0](https://wso2.com/identity-and-access-management/), all the UIs are
+present in **authenticationendpoint **and **accountrecoveryendpoint **web apps
+in <IS-Home>/repository/deployment/server/webapps folder. Please follow the
```
Comment (YvonneW) [inline en/docs/develop/customizing-user-interfaces.md]: User monospace when referring to folder paths and file names.

File: en/docs/develop/customizing-user-interfaces.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+Each web app(authenticationendpoint and recoveryendpoint), contains a folder called "includes" which contains
+header.jsp and footer.jsp file. Copy the header.jsp and footer.jsp files in includes to extensions folder created in Step 1.
+ The header.jsp file and footer.jsp file has the capability to override the files in "includes" folder.
+
+#### Step 3: Create basic CSS styles need for UI
+
+Create the styles that need to be override.
+
+    <style type="text/css">
```
Comment (YvonneW) [inline en/docs/develop/customizing-user-interfaces.md]: Use code block syntax.

```
```

File: en/docs/develop/customizing-identity-server-uis.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+All these UIs are available in the **`authenticationendpoint`** and **`accountrecoveryendpoint`** web apps that are located in the `<IS-Home>/repository/deployment/server/webapps` folder. To customize these pages, follow the steps below. 
+
+
+## Step 1: Create the extensions folder
+
+1. Navigate to the `<IS-Home>/repository/deployment/server/webapps/authenticationendpoint` folder and create a new folder called **`extensions`**.
+
+!!! info
+
+    You will be adding the customized JSP files along with the custom CSS files into this new `extensions` folder.
```
Comment (YvonneW) [inline en/docs/develop/customizing-identity-server-uis.md]: Fix indentation.

File: en/docs/develop/customizing-identity-server-uis.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+
+       Make sure to add the following include to the top, when editing the `header.jsp` file of the accountrecovery web application.
+
+       ```
+       <%@include file=”../localize.jsp” %>
+       ```
+
+4. Refresh the browser and check out the customized header and footer content. 
+
+!!! tip
```
Comment (YvonneW) [inline en/docs/develop/customizing-identity-server-uis.md]: Fix indentation.

**Pull request merged**